giskard/core/savable.py

```diff
@@ -157,7 +157,7 @@ def download(cls, uuid: str, client: Optional[GiskardClient], project_key: Optio
 class RegistryArtifact(Artifact[SMT], ABC):
     def _save_locally(self, local_dir: Path):
         with open(Path(local_dir) / "data.pkl", "wb") as f:
-            cloudpickle.dump(self, f, protocol=pickle.DEFAULT_PROTOCOL)
+            dump_by_value(self, f)
```

Comment:
@kevinmessiaen we should use `dump_by_value` in `CloudpickleSerializableModel.save_model` too, shouldn't we?

Reply:
probably in `save_model_class` too just to centralize all pickling in one place

Reply:
Yes that's right, I'll add this modification